.github/workflows/build.yml

```diff
@@ -15,12 +15,19 @@ jobs:
         os:
           - fedora:latest
           - quay.io/centos/centos:stream8
-          - quay.io/centos/centos:stream9
```

Comment:
Is this removal necessary?

Reply:
It's in the commit message.  CentOS 7 and Stream 9 got moved out of the first block (which uses both clang and gcc) and into the block below configured simply for gcc.  clang wasn't getting installed correctly on either CentOS 7 nor Stream 9 and although it's probably some rpm/dnf magic I'm overlooking I couldn't figure it out after a little looking so it seemed better to (for the moment at least) just drop the clang compiles to get the CI tests running and hopefully someone can figure out what needs to happen to get clang happy on those two OSs.

Reply:
Hello. Thanks for clarifying. I will open an Issue then to track the clang on CentOS Stream 9 issue.

Reply:
I'm not sure why clang in Stream 9 was failing (possibly getting wrong install dependencies section) but it works now with the cleaned up version I just pushed. Only CentOS 7 seems to fail with clang and that's because clang is in the SCL in CentOS 7 and would require changes to the build to run the compiler with the SCL command.  Since it's clearly never worked before and the gcc build works fine and CentOS 7 is approaching EOL, it doesn't seem worth the work to make the changes necessary for clang rather than just live with gcc there.

Reply:
Hello @hdholm. Thanks for the PR. I agree it is not that important to have clang compilation in CentOS7 right now.
In my opinion, this can be merged.

Let's wait a little bit for @sergio-correia to have a view on it and we will merge it before creating the new release.